src/test/scala/ai/privado/languageEngine/java/tagger/collection/MethodFullNameCollectionTaggerTest.scala

```diff
@@ -0,0 +1,109 @@
+package ai.privado.languageEngine.java.tagger.collection
+
+import ai.privado.languageEngine.java.AbstractTaggingSpec
+import ai.privado.model.*
+import io.shiftleft.codepropertygraph.generated.Cpg
+import io.shiftleft.semanticcpg.language.*
+
+class MethodFullNameCollectionTaggerTest extends AbstractTaggingSpec(language = Language.JAVA) {
+  val collectionRule: RuleInfo = RuleInfo(
+    "Collections.Spark.HttpFramework",
+    "Spark Java Http Framework Endpoints",
+    "",
+    FilterProperty.METHOD_FULL_NAME,
+    Array(),
+    List(".*\\b(get|post|put)\\b.*"),
+    false,
+    "",
+    Map(),
+    NodeType.REGULAR,
+    "",
+    CatLevelOne.COLLECTIONS,
+    catLevelTwo = Constants.default,
+    Language.JAVA,
+    Array()
+  )
+
+  "Spark Http Framework" should {
+    "tag collection endpoint when handler is a method" in {
+      var cpg: Cpg = null
+      try {
+        val javaFileContents: String =
+          """
+            |import static spark.*;
+            |
+            |public class HelloWorld {
+            |    public static void main(String[] args) {
+            |        Spark.get("/hello", (req, res) -> "Hello World");
+            |    }
+            |}""".stripMargin
+        cpg = buildCpg(javaFileContents)
+        cpg.call.methodFullName(".*get.*").l.size shouldBe 1
+        cpg.call.head.code.contains("get") shouldBe true
```

Comment:
This is not a good practise, as if any change in AST creator which modifies the order of call nodes, can result into this test case failure.

we can do
```
val getCalls = cpg.call.methodFullName(".*get.*").l
getCalls.size shouldBe 1
getCalls.name shouldBe List("get")
```